app/views/helpers/feed/update.phtml

```diff
@@ -274,7 +274,11 @@
 		<div class="form-group">
 			<label class="group-name" for="path_entries"><?= _t('sub.feed.css_path') ?></label>
 			<div class="group-controls">
-				<input type="text" name="path_entries" id="path_entries" class="extend" value="<?= $this->feed->pathEntries() ?>" placeholder="<?= _t('gen.short.blank_to_disable') ?>" />
+				<div class="stick">
+					<input type="text" name="path_entries" id="path_entries" class="extend" value="<?= $this->feed->pathEntries() ?>" placeholder="<?= _t('gen.short.blank_to_disable') ?>" />
+					<a id="popup-preview-selector" class="btn" href="<?= _url('feed', 'contentSelectorPreview', 'id', $this->feed->id(), 'selector', 'selector-token') ?>"><?= _i('look') ?></a>
+				</div>
+				<br/>
```

Comment:
Why not, but my intend was, in fact, to open another PR once this one would be validated (?) to fix other fields and make things coherent. Because the current results (help on the same line) can look really broken:
<img width="477" alt="Screenshot 2020-01-30 at 22 12 46" src="https://user-images.githubusercontent.com/1971659/73490956-dd559c80-43ad-11ea-97ea-f0b0d94aaae7.png">

Note, thought, that some existing fields already put the help on a new line:
<img width="512" alt="Screenshot 2020-01-30 at 22 13 07" src="https://user-images.githubusercontent.com/1971659/73490984-eba3b880-43ad-11ea-8db5-b4b41e5e76f6.png">

Reply:
Here is the result without the `<br/>` :s

<img width="490" alt="Screenshot 2020-01-30 at 22 50 13" src="https://user-images.githubusercontent.com/1971659/73493415-fdd42580-43b2-11ea-8a61-353c33e8aa4b.png">

Reply:
Fixed, by the way ;)